### PR TITLE
Site css update

### PR DESCRIPTION
**Pull request by ctwoolsey**

### CODE REVIEW

File: app/scripts/controllers/courtSearchInfo.ctrl.js [hunk 1]
```diff
@@ -11,7 +11,11 @@ angular.module('yourStlCourts').controller('CourtSearchInfoCtrl', function ($sta
     zoom: 14
   };
 
-  ctrl.courtMarkers = {};
+  leafletData.getMap("mapid").then(function(map){
```
Comment (abobwhite) [inline app/scripts/controllers/courtSearchInfo.ctrl.js]: This doesn't seem appropriate in a controller. Can this just be set in the HTML or somewhere else?

File: app/scripts/controllers/footer.ctrl.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('yourStlCourts').controller('FooterCtrl', function () {
+  var ctrl = this;
+
+  ctrl.copyRightYears = function(startingYear){
```
Comment (abobwhite) [inline app/scripts/controllers/footer.ctrl.js]: The starting year doesn't really need to be a param. It's got to be hard-coded anyway, so might as well explicitly set it here in the controller.

File: app/views/courtSearchInfo.html [hunk 3]
```diff
@@ -1,34 +1,37 @@
...
+      <div class="col-sm-6 alignColumnsToPageToPage">
+        <h2 class="no-indent">Court Address</h2>
+        <p class="address no-indent">{{ctrl.courtInfo.address}}</p>
+        <p class="address no-indent">{{ctrl.courtInfo.city}}, {{ctrl.courtInfo.state}} {{ctrl.courtInfo.zip_code}}</p>
 
-            <div ng-if="ctrl.courtWebsite != '' ">
-              <h3>Website</h3>
-              <p ng-bind-html="ctrl.courtWebsite | linky"></p>
-            </div>
+        <div class="topMargin" ng-if="ctrl.courtPhone != '' ">
```
Comment (abobwhite) [inline app/views/courtSearchInfo.html]: In JavaScript, empty string is falsy. you can change these ng-if's to just "!"....e.g. `ng-if="ctrl.courtPhone"`....for all of them

File: app/scripts/controllers/courtSearchInfo.ctrl.js (hunk 1 above)
Comment (ctwoolsey) [inline app/scripts/controllers/courtSearchInfo.ctrl.js]: This is the only way to get the map object and then turn off it's dragging.  I tried making it an option with the code above it, but it doesn't seem to get read in that way at least when we use ui-leaflet.  If we just used Leaflet, then I might be able to initialize it on map creation.

Comment (abobwhite) [inline app/scripts/controllers/courtSearchInfo.ctrl.js]: OK for now

**Pull request merged**